apis/core.oam.dev/v1alpha2/application_types.go

```diff
@@ -56,6 +56,9 @@ type AppStatus struct {
 	// Services record the status of the application services
 	Services []ApplicationComponentStatus `json:"services,omitempty"`
 
+	// ResourceTracker record the status of the ResourceTracker
+	ResourceTracker *runtimev1alpha1.TypedReference `json:"resource_tracker,omitempty"`
```

Comment:
```suggestion
	ResourceTracker *runtimev1alpha1.TypedReference `json:"resourceTracker,omitempty"`
```

We rarely use `xxx_yyy` style in K8s CRD

Reply:
Got it